lib/notion-sdk-ruby/request_client.rb

```diff
@@ -4,6 +4,7 @@ class RequestClient
 
     base_uri "https://api.notion.com"
     headers "Content-Type": "application/json"
+    headers "Notion-Version": "2021-05-13"
```

Comment:
Let's default the version to `"2021-05-13"` but also add `version` to `Notion.config` so that it can be configured by the client. The implementation should be very similar to the `api_token`.

Reply:
Thanks for cleaning up, @mgmarlow! Sorry to be a drive-by committer :)

Reply:
Hey, no problem! Thanks for filling the issue and opening a PR.